otj-server-reactive/src/test/java/com/opentable/server/reactive/TestReactiveServerConfiguration.java

```diff
@@ -55,15 +54,32 @@ WebClient webClient() {
         return WebClient.builder().build();
     }
 
+    /**
+     * This scheduler is used by {@link ConservedHeadersWebFilterTest}
+     * to set up fewer available threads than the number of requests we want to make in parallel, to ensure that
+     * reactive operations from different requests are scheduled on threads that were previously used by another
+     * request.
+     */
+    @Bean
+    Scheduler getScheduler() {
+        return Schedulers.newParallel("parallel-scheduler", 3);
```

Comment:
ignorant: any reason you chose this scheduler as oppoosed to the gajillion others (the size I understand)

Reply:
Relatively arbitrary. It creates a new fixed-size scheduler that drops in to the Reactor scheduler interface.